tests/general/CasingBearTest.py

```diff
@@ -0,0 +1,98 @@
+from queue import Queue
+from bears.general.CasingBear import CasingBear
+from bears.general.AnnotationBear import AnnotationBear
+from tests.LocalBearTestHelper import LocalBearTestHelper
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.bears.LocalBear import LocalBear
+
+
+class CasingBearTest(LocalBearTestHelper):
+
+    def setUp(self):
+        self.section = Section("test section")
+
+    def get_results(self, file, section=None):
+        self.dep_uut = AnnotationBear(section, Queue())
+        dep_results_valid = self.dep_uut.execute("file", file)
+        uut = CasingBear(section, Queue())
+
+        if dep_results_valid == None:
```

Comment:
this will never happen the case of `AnnotationBear` even if it there is no result it still yields a hiden results with empty contents, if it returns None it means there was an error, which you should take care of.